### PR TITLE
Quieter CI test logs

### PR DESCRIPTION
**Pull request by snickell**
Our CI test logs have many fairly extraneous lines, many commands in `docker/ui_tests.sh` and `docker/unit_tests.sh` are run with explicitly increased (and pretty extreme) verbosity. This probably made more sense when they were last seriously edited (several years ago, basically when they were being created) because the scripts themselves were under heavy debugging. But at this point, its adding huge amounts of un-read noise that has to be skipped to find the real errors.

- Previously ui tests output (success/fail) in drone **started on line ~8000**. Now it **starts on line 100**.
- As a rule, flags were selected that ensure error output was NOT suppressed.

Outline of CI log output modified:
- about a thousand lines of webpack stats about the bundling and splitting of assets has been supressed, error stats are maintained
- a several thousand line list of every file auto-output by `rake assets:precompile` has been supressed, errors are maintained
- the `set -x` command that echos every shell command run has been moved lower in the script to where it feels relevant to tracking what step is being run: "now I'm running bundle install", vs echoing a few hundred lines of shell noise
- noisy timestamps only on the unit tests (but not UI test) have been removed, this complicated the file, added "half a screen width of noise" to a random assortment of CI log outputs, and unit tests are probably not the place where timing is super useful anyway
- `Flushing Cdo::Metrics::Buffer, waiting Infinity seconds` messages are interspersed frequently throughout CI test output, making it harder to scan, these have been changed to debug loglevel messages (checked months ago if these were helpful, and pablo agreed they weren't)
- long list of locals.yml embedded in `unit_tests.sh` and `ui_tests.sh` was sorted alphabetically just to clean up the file a tiny bit

### CODE REVIEW

**Comment by snickell** [comment]
(drone still running but previous failures are cancelled builds, latest drone run is already succesfully past the point in the scripts that I modified)

File: docker/ui_tests.sh [hunk 1]
```diff
@@ -4,7 +4,9 @@
 # In most cases, you will not run this script directly, but instead
 # use docker-compose to run using the ui-tests-compose.yml file in this directory. See instructions in that file.
 
-set -xe
+# rerun tests 1
```
Comment (molly-moen) [inline docker/ui_tests.sh]: should this comment be here?

**Review by molly-moen — APPROVED** [review]
This seems reasonable to me! Might be good to have a sanity check from @code-dot-org/infrastructure too.

**Review by cat5inthecradle — APPROVED** [review]

**Comment by cat5inthecradle** [comment]
Same nit about `rerun tests 1` as Molly, but otherwise LGTM.

File: docker/ui_tests.sh [hunk 2]
```diff
@@ -26,60 +28,54 @@ mkdir $CIRCLE_ARTIFACTS
 # rbenv-doctor https://github.com/rbenv/rbenv-installer#readme
 curl -fsSL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-doctor | bash
 
-bundle install --verbose
-
 # set up locals.yml
 # Need to actually write all the commented out lines also
 set +x
```
Comment (wilkie) [inline docker/ui_tests.sh]: technically, it is already `+x` since you removed the `-x` earlier.

File: docker/unit_tests.sh [hunk 3]
```diff
@@ -4,50 +4,41 @@
...
 export CI=true
 export RAILS_ENV=test
 export RACK_ENV=test
 export DISABLE_SPRING=1
 export LD_LIBRARY_PATH=/usr/local/lib
 
-mispipe "bundle install --verbose" ts
-
 # set up locals.yml
 set +x
```
Comment (wilkie) [inline docker/unit_tests.sh]: Ditto, here, I believe. Redundant after removing `-x` earlier.

**Review by wilkie — APPROVED** [review]
I love it! Thank absolute goodness!! 🎉 Minor nits about the `set +x`... either place it at the beginning or remove them where they happen after a place where you've removed the `set -x` preceding it.

**Pull request merged**